### PR TITLE
Fix API documentation responses [CLOUDDST-2927]

### PR DESCRIPTION
**Pull request by pbortlov**

### CODE REVIEW

File: iib/web/static/api_v1.yaml [hunk 1]
```diff
@@ -181,6 +186,7 @@ paths:
           application/json:
             schema:
               # TODO: update to include Regenerate Bundle attrs
```
Comment (pbortlov) [inline iib/web/static/api_v1.yaml:188]: Should we add regenerate-bundle and merge-index-image?

File: iib/web/static/api_v1.yaml [hunk 2]
```diff
@@ -331,12 +337,12 @@ paths:
     post:
       description: Submit a build request to regenerate an operator bundle image
       requestBody:
-        description: The build request to create
+        description: The build request to create #TODO add description???
```
Comment (pbortlov) [inline iib/web/static/api_v1.yaml:340]: Aany idea?

**Comment by pbortlov** [comment]
@yashvardhannanavati  @lipoja I suggest create path '#/components/schemas/response/' and '#/components/schemas/request'

File: iib/web/static/api_v1.yaml [hunk 3]
```diff
@@ -862,21 +944,7 @@ components:
...
-            from_bundle_image:
-              type: string
-              description: The pull specification of the original bundle image that was modified.
-              example: 'quay.io/user/openshift-bundle-example-container:v1'
-            organization:
-              type: string
-              description: The name of the organization the bundle was regenerated for.
-              example: openshift-operators
-    RequestUpdate:
+    RequestUpdate: #TODO shouldn't be just add/rm ??? Is it request or response?
```
Comment (pbortlov) [inline iib/web/static/api_v1.yaml:947]: @yash what request could be updated?

File: iib/web/static/api_v1.yaml (hunk 1 above)
Comment (pbortlov) [reply iib/web/static/api_v1.yaml:188]: @yashvardhannanavati could you please validate what belongs here?

File: iib/web/static/api_v1.yaml [hunk 4]
```diff
@@ -132,8 +136,9 @@ paths:
             application/json:
               schema:
                 oneOf:
-                  - $ref: '#/components/schemas/RequestIndexImageVerbose'
-                  - $ref: '#/components/schemas/RequestRegenerateBundleVerbose'
+                  - $ref: '#/components/schemas/AddResponseVerbose'
+                  - $ref: '#/components/schemas/RmResponseVerbose'
+                  - $ref: '#/components/schemas/RegenerateBundleResponseVerbose' #TODO do other belongs here?
```
Comment (yashvardhannanavati) [inline iib/web/static/api_v1.yaml:141]: I don't think others belong here. It should be safe to remove the TODO

File: iib/web/static/api_v1.yaml (hunk 1 above)
Comment (yashvardhannanavati) [reply iib/web/static/api_v1.yaml:188]: yup, we will have to add the attributes that we call the `patch` endpoint with from the regenerate-bundle and merge-index-image endpoints. I believe they can be found in `build.py` and `build_merge_index_image.py`

File: iib/web/static/api_v1.yaml [hunk 5]
```diff
@@ -626,16 +663,13 @@ components:
...
-              type: string
-              example: openshift-operators
             removed_operators:
               type: array
               items:
                 type: string
               example:
                 - prometheus-operator
-    RequestAdd:
+    AddRequest: # TODO maybe optimalize Add and Rm
```
Comment (yashvardhannanavati) [inline iib/web/static/api_v1.yaml:672]: Can I ask optimize in what sense?

File: iib/web/static/api_v1.yaml (hunk 3 above)
Comment (yashvardhannanavati) [reply iib/web/static/api_v1.yaml:938]: This is the schema used in the `requestBody` of the `patch` endpoint. It's a request if I am not wrong

File: iib/web/static/api_v1.yaml (hunk 5 above)
Comment (pbortlov) [reply iib/web/static/api_v1.yaml:672]: Add and RM request use similar strucuture. But on the second thought, I think we don't need to do anything with that.

File: iib/web/static/api_v1.yaml (hunk 4 above)
Comment (pbortlov) [reply iib/web/static/api_v1.yaml]: removed

**Comment by pbortlov** [comment]
@yashvardhannanavati I can find build.py nor build_merge_index_image.py. Could you point me to the files or project?

**Comment by pbortlov** [comment]
PTAL @yashvardhannanavati @lipoja .. It seems there is a problem with Travis today.

**Review by lipoja — APPROVED** [review]

**Review by yashvardhannanavati — APPROVED** [review]

**Pull request merged**